riscv-asm.md

```diff
@@ -525,6 +528,82 @@ Integers in the subinterval [-2048, -1] can also be passed by their (unsigned) a
+0.5                         | 0x1p-1                         | 0.5                   | 5.000000e-01
+0.625                       | 0x1.4p-1                       | 0.625                 | 6.250000e-01
+0.75                        | 0x1.8p-1                       | 0.75                  | 7.500000e-01
+0.875                       | 0x1.cp-1                       | 0.875                 | 8.750000e-01
+1.0                         | 0x1p+0                         | 1.0                   | 1.000000e+00
+1.25                        | 0x1.4p+0                       | 1.25                  | 1.250000e+00
+1.5                         | 0x1.8p+0                       | 1.5                   | 1.500000e+00
+1.75                        | 0x1.cp+0                       | 1.75                  | 1.750000e+00
+2.0                         | 0x1p+1                         | 2.0                   | 2.000000e+00
+2.5                         | 0x1.4p+1                       | 2.5                   | 2.500000e+00
+3                           | 0x1.8p+1                       | 3.0                   | 3.000000e+00
+4                           | 0x1p+2                         | 4.0                   | 4.000000e+00
+8                           | 0x1p+3                         | 8.0                   | 8.000000e+00
+16                          | 0x1p+4                         | 16.0                  | 1.600000e+01
+128 (2 ^ 7)                 | 0x1p+7                         | 128.0                 | 1.280000e+02
+256 (2 ^ 8)                 | 0x1p+8                         | 256.0                 | 2.560000e+02
+2 ^ 15                      | 0x1p+15                        | 32768.0               | 3.276800e+04
+2 ^ 16                      | 0x1p+16                        | 65536.0               | 6.553600e+04
+Positive infinity           | inf                            | inf                   | inf
+Canonical NaN               | nan                            | nan                   | nan
+
+
+The `fli` instruction is used to load a floating point immediate into a floating
+register. The floating point immediate must be one of the values listed in the
+above table and can be expressed in hexadecimal, decimal, or standard notation
+(also known as scientific notation).
+
+```assembly
+	fli.s	fa0, 0x1p-15
+	fli.s	fa1, 0.00390625
+	fli.s	fa2, 6.250000e-02
+```
+
+RISC-V does not offer a generic pseudoinstruction to load an arbitrary floating
+point immediate value. Instead, a programmer can use the `.float`/`.double`
+directive to declare a floating point immediate value in the source code, and
+then load it into a floating point register using the load global
+pseudoinstruction (`fl{h|w|d|q}`).
+
+RISC-V didn't provide generic pseudoinstruction to load arbitrary floating point
+immediate value, programmer can use `.float`/`.double` diretive to declare a
+float point immediate within the source, and then load into floating point
```

Comment:
float -> floating

Reply:
and the load into -> and then load it into a

Reply:
Same here